packages/lodestar/src/api/impl/validator/index.ts

```diff
@@ -166,6 +171,20 @@ export function getValidatorApi({chain, config, logger, metrics, network, sync}:
       );
   }
 
```

Comment:
Maybe we need some comments here regarding why we calculate it here instead of the state transition

Reply:
I am not sure I follow, are you referring to adding a comment to `getGenesisBlockRoot` function?

Reply:
nvm, this function just calls `computeProposers` without adding more logics